index.html

```diff
@@ -5,30 +5,28 @@
   <meta charset="UTF-8" />
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link href="assets/CSS/style.css" rel="stylesheet" />
   <link href="assets/CSS/desktop.css" rel="stylesheet" />
+  
+
   <title>Portfolio</title>
 </head>
 <body>
   <header>
     <div class="flex-container">
       <nav class="logo">
-        <h2>
-          <a href="#">My Logo</a>
-        </h2>
-        <img src="assets/images/Icon-Menu.jpg" alt="Icon menu" class="to-display"/>
-        <ul class="to-hide">
-          <li>
-            <a href="#">Portofolio</a>
-          </li>
-          <li>
-            <a href="#">About</a>
-          </li>
-          <li>
-            <a href="#">Contact</a>
-          </li>
-        </ul>
+          <a href="#" class="nav-brand">My Logo</a>
+          <ul class="list-nav ">
+            <li class="nav-item"><a href="#" class="nav-link">Portfolio</a></li>
+              <li class="nav-item"><a href="#"class="nav-link">About</a></li>
+              <li class="nav-item"><a href="#" class="nav-link">Contact</a></li>
```

Comment:
- [ ] These links that come up when the menu bar is clicked should link to the appropriate section of the webpage. For example the `Portfolio` link should link to the top of the index page, the `About` should go to the about section when clicked and the `Contact` should go to the contact section when clicked. Kindly fix this issue as it is part of the requirements.
![image](https://user-images.githubusercontent.com/81932478/174863390-55b41ba8-3ab9-42a3-81d8-7f6898ec7e92.png)

Reply:
solved